app/themes/dashboard/ui/index.html

```diff
@@ -0,0 +1,116 @@
+<!doctype html>
+<html>
+  <head>
+    <base href="/">
+    <meta charset="utf-8">
+    <title>@sample-app-name</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width">
+    <link rel="shortcut icon" href="/images/favicon.ico" />
+
+    <!-- build:css styles/lib.css -->
+    <!-- bower:css -->
+    <!-- endbower -->
+    <!-- endbuild -->
+
+    <!-- build:css styles/app.css -->
+    <!-- inject:css -->
+    <!-- endinject -->
+    <!-- endbuild -->
+
+  </head>
+  <body ng-app="app" class="skin-black">
+
+    <div class="wrapper">
+      <header class="main-header">
+
+        <!-- Logo -->
+        <a ui-sref="home" class="logo">
+          <!-- mini logo for sidebar mini 50x50 pixels -->
+          <span class="logo-mini"><b>ML</b></span>
+          <!-- logo for regular state and mobile devices -->
+          <span class="logo-lg"><h1 id="logo">Sample App</h1></span>
+        </a>
+
+        <nav class="navbar navbar-static-top" role="navigation" ng-controller="topnavCtrl as topnav" ng-cloak>
+          <!-- Navbar Right Menu -->
+          <div class="navbar-custom-menu">
+            <ul class="nav navbar-nav" ng-if="!topnav.currentUser">
+              <li class="user">
+                <a ui-sref="root.login"><span class="glyphicon glyphicon-user"></span> Sign In</a></li>
+            </ul>
+            <ul class="nav navbar-nav" ng-if="topnav.currentUser">
+              <li>
+                <a ui-sref="dashboard.overview">
```

Comment:
Hey when I clicked this link it didn't resolve to anything, don't think a page exists for it, would it make sense for it to be `ui-sref="root.landing"` ?